src/Contracts/HasAbilities.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Butler\Auth\Contracts;
+
+interface HasAbilities
+{
+    public function can(string $ability): bool;
+    public function cant(string $ability): bool;
```

Comment:
```suggestion
    public function cannot(string $ability): bool;
```

To match Laravel?

Reply:
https://github.com/laravel/sanctum/blob/2.x/src/Contracts/HasAbilities.php#L21

Reply:
https://laravel.com/docs/8.x/authorization#via-the-user-model

🤷

Reply:
Updated.